zenodo/modules/records/templates/zenodo_records/update_datacite.html

```diff
@@ -0,0 +1,86 @@
+{#
+# This file is part of Zenodo.
+# Copyright (C) 2016 CERN.
```

Comment:
Should be 2018. Take a look at other files as well that you either created or modified.